ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -20,7 +20,7 @@
 #
 
 function usage() {
-  echo "USAGE: enable-ranger-plugin.sh -r ranger_host:ranger_port -u ranger_user -p ranger_password [-h hawq_host:hawq_port] -w hawq_user -q hawq_password"
+  echo "USAGE: enable-ranger-plugin.sh -r ranger_host:ranger_port -u ranger_user -p ranger_password [-h hawq_host:hawq_port -c hawq_kerberos_service_name] -w hawq_user -q hawq_password -t lookup_authentication_type"
   exit 1
 }
```

Comment:
I think "-t lookup_authentication_type" should be moved in front of "-c hawq_kerberos_service_name", and should also be marked as optional with default value "simple". 
"-c hawq_kerberos_service_name" can be changed to "-s", since it indicates the service name.